testsuite/tests/singlecluster/limitador/tokens_rate_limit/test_trlp.py

```diff
@@ -0,0 +1,78 @@
+"""Test TokenRateLimitPolicy functionality"""
+
+import pytest
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador, pytest.mark.authorino]
+
+
+@pytest.fixture(autouse=True)
+def add_opa_policy(authorization):
```

Comment:
wrong module for opa policy. And if you'll name it the same as authorization fixture, it will be picked up without autouse